client/src/app/process-details/process-details.component.html

```diff
@@ -105,7 +105,7 @@ <h4>Protocols</h4>
               <p><strong>name: </strong> {{protocol?.content['protocol_core']['protocol_name']}}</p>
               <p><strong>description: </strong> {{protocol?.content['protocol_core']['protocol_description']}}</p>
             </mat-card-content>
-            <mat-icon-button (click)="removeProtocol(protocol)" style="cursor: pointer" matTooltip="Remove Protocol">
+            <mat-icon-button (click)="removeProtocol(protocol)" style="cursor: pointer" matTooltip="Unlink Protocol">
               <mat-icon>delete</mat-icon>
```

Comment:
is it possible to use a - or x icon instead of bin icon? bin icon might look like we're deleting the entity

Reply:
i dont know actually, we can just wait on their feedback as well.

Reply:
Good point but yeah let's wait for feedback